src/application/can_handler/can_handler.c

```diff
@@ -0,0 +1,80 @@
+#include "can_handler.h"
+
+QueueHandle_t busQueue_rx;
+QueueHandle_t busQueue_tx;
```

Comment:
```suggestion
QueueHandle_t busQueue_rx = NULL;
QueueHandle_t busQueue_tx = NULL;
```
as per BARR-C:2018 7.2d: "Any pointer variable lacking an initial address shall be initialized to `NULL`."

Also, these variable names aren't exactly `snake_case`.